meteoblue_dataset_sdk/caching/filecache.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import hashlib
+import json
+import logging
+import os
+import tempfile
+import zlib
+from typing import Union, Optional
+
+import aiofiles
+from aiofiles import os as aios
+
+from .cache import Cache
+
+CACHE_DIR = "mb_cache"
+# 7200s == 2h
+DEFAULT_CACHE_DURATION = 7200
+
+
+class FileCache(Cache):
+    def __init__(
+        self,
+        cache_path: Optional[str] = None,
+        cache_ttl: int = DEFAULT_CACHE_DURATION,
+        compression_level: int = 6,
```

Comment:
the  prefix `cache_` for those paramater is not required. I would not use `ttl`, because files are not deleted. Please use `max_age` as specified.